### ISSUE TITLE
azure-themes light mode has insufficient contrast in DetailsList

### ISSUE THREAD

**Issue by raharri** [microsoft/fluentui#18380]
### Environment Information

- **Package version(s)**: azure-themes 8.1.16
- **Browser and OS versions**: Chrome  91.0.4472.77 for Windows 10

### Describe the issue:

The color contrast between a Link element and the background of a selected row in a DetailsList is insufficient (ratio of #0179D4 to #F3F2F1 is 4:1 when it should be 4.5:1)

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/raharri/pen/RwpLBBy

#### Actual behavior:

![image](https://user-images.githubusercontent.com/57734445/120052432-2cc7cf00-bfda-11eb-9228-4b799f954841.png)

#### Expected behavior:

The color contrast should be at least 4.5:1

### Documentation describing expected behavior

https://webaim.org/articles/contrast/#sc143

**Comment by Jacqueline-ms** [comment]
Light theme: new selected row link color: #106EBE per design files. Same color for selected hover row link. 

![image](https://user-images.githubusercontent.com/30805892/121085066-43b6b000-c796-11eb-8bfc-86b6df3020f2.png)


Contrast now passes accessibility at 4.7:1

![image](https://user-images.githubusercontent.com/30805892/121085377-9ee8a280-c796-11eb-846e-0c1c3236dc09.png)

Currently checking on the selected + hover states.
